### DISCUSSION TITLE
Create a Q&A section inside the documentations for each endpoint.

### DISCUSSION THREAD

**Discussion by Makebuildmodify** [etsy/open-api#995] [Ideas]
Wouldn't is be nice if, inside the documentation, there was a Q&A section for each endpoint for commonly asked question?  This would be a toggle to open section for each endpoint. I wouldn't interfere with the current look of the documentations, but it would be nice to toggle it to get examples and answers to the simple questions. It would save tones of time.

**Comment by Gareth064** [comment]
I have been thinking for a while on how we, as a community, could put together a FAQ somewhere. Preferably if would be here in this GitHub.

But we'd have to all agree on the format, and whether there would be code examples, if so then in which language etc etc.
